doc/sphinx/source/recipes/recipe_cvdp.rst

```diff
@@ -36,10 +35,7 @@ Variables
 Observations and reformat scripts
 ---------------------------------
 
-*Note: (1) obs4mips data can be used directly without any preprocessing;
-(2) see headers of reformat scripts for non-obs4mips data for download
-instructions.*
-
+None. 
```

Comment:
Why are there no observations? Comparison with observation is one of the main aspects of the CVDP. To me, tis looks like an incomplete draft implementation of the CVDP into the ESMValTool.